examples/advanced/flutter/.dart_tool/package_config.json

```diff
@@ -0,0 +1,176 @@
+{
```

Comment:
I think this file is generated by flutter; it contains a lot of paths to your local machine. I'm not 100% sure, but I think it should not be included.

Some info I found on what to commit/not commit:
https://dart.dev/guides/libraries/private-files

Reply:
The .dart_tool is generated on pub get, so it's safe to delete